terraform/aws/public-cloud/main.tf

```diff
@@ -35,6 +37,16 @@ module "aws-keypair" {
   public_key_filename = "${var.public_key_file}"
 }
 
+module "ca" {
+  source            = "github.com/Capgemini/tf_tls//ca"
```

Comment:
this is the proper syntax as per here when using subdirectories https://www.terraform.io/docs/modules/sources.html